example/lib/src/screens/my_home_page.dart

```diff
@@ -19,7 +19,7 @@ class _MyHomePageState extends State<MyHomePage> {
 
   final primaryColorController = TextEditingController();
   final screenNameController = TextEditingController();
-
+  final featureFlagsController = TextEditingController();
```

Comment:
I can't find the `TextEditingController` dispose function, can we please add it if it's not exist?

Reply:
done